nservicebus/sqlserver/configuration.md

```diff
@@ -46,6 +43,68 @@ By passing the transport a custom factory method which will provide connection s
 
 snippet:sqlserver-custom-connection-factory
 
+## Multiple connection strings
+
+In [*multi-catalog* and *multi-instance* modes](/nservicebus/sqlserver/deployment-options.md) it is necessary to include additional information in configuration.  The sender needs to know the connection string of the receiver, and the subscriber needs the connection string of the publisher.
+
+Connection strings for the remote endpoint can be configured in several ways:
+
+### Via the App.Config
+
+In Versions 2.1 to 2.x the endpoint-specific connection information is discovered by reading the connection strings from the configuration file with `NServiceBus/Transport/{name of the endpoint in the message mappings}` naming convention. If such a connection string is found, it is used for a given endpoint and this setting has precedence over the code-provided connection information.
+
+Starting from Version 3.0, the connection string for a remote endpoint has to be provided using configuration API.
```

Comment:
Should be Versions 3 and above.
See version naming convention here: https://github.com/Particular/docs.particular.net#version-language